### PR TITLE
make_deb automated to check underlying arch and remove preinst

### PR DESCRIPTION
**Pull request by Singhal-Vaibhav**
**Changes**:
1. creation of debian packages checks for underlying architecture.
2. Added dependency of build-essential,python3-dev in control file since it is required to build Adafruit_Python_DHT
3. removed preinst for debian packages, not required.

**Note**
The dependency of Adafruit_Python_DHT is not a dependency to create debian package, its a runtime dependency and hence make_deb not modified to clone this repo and build. Also to build this dependency, it requires sudo access which make_deb does not have.

### CODE REVIEW

**Comment by praveen-garg** [comment]
We should merge this PR https://github.com/foglamp/foglamp-south-dht11/pull/8 first.

**Comment by praveen-garg** [comment]
@Singhal-Vaibhav  please rebase this PR.

File: make_deb [hunk 1]
```diff
@@ -16,67 +16,51 @@
...
 ##
-## Author: Ivan Zoratti, Praveen Garg
+## Author: Ivan Zoratti, Praveen Garg, Vaibhav Singhal
 ##
 
 set -e
 
 GIT_ROOT=`pwd`    # The script must be executed from the root git directory
-architecture="none"
-usage="$(basename "$0") {arm} [clean|cleanall]
```
Comment (praveen-garg) [inline make_deb:26]: restore usage, just remove arm

help option needs it. may be help should also have `-h` as `-h | help`

**Review by praveen-garg — CHANGES_REQUESTED** [review]

File: make_deb (hunk 1 above)
Comment (Singhal-Vaibhav) [reply make_deb:26]: done

**Review by praveen-garg — APPROVED** [review]

**Pull request merged**